### ISSUE TITLE
"single word" can't take you out of context

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#38]
“1. On all question types, a quizzer cannot automatically be ruled out of context by a single word, even if it is a unique word.  Rather, the quizzer must say a complete thought or phrase that conclusively puts them out of context as ruled by the answer judge.”

Clarify this.    I have heard it misapplied.  For example, if the start of one verse is “I am the [unique word A]” and the start of another verse is “I am the [unique word B]”, and the quizzer started in one verse but then switched to the other, that they cannot be counted out of context because of the above rule.   That was never the intention of the rule.   In this example, the quizzer has quoted the first four words from a different verse, which puts them out of context.  You cannot look at the one different word in a vacuum. 

The rule was written so that if a quizzer is in the correct context, but happens to say a word that is out of context, they cannot be counted out of context due to that one word.

**Comment by gryphonshafer** [comment]
This is an iceberg topic. Seems small on the surface, but there are huge implications below the surface. When we get to this one, we should call for the Second Ecumenical Council.

**Comment by scottpeterson** [comment]
Definitely. It will requiring taking each Question Type separately, and asking + answering the question "what does the existence of "context" get us, what is the ultimate goal?"

The quizzer "not being incorrect" is a much more objective goal, and covers the vast majority of "quizzer answering scenarios."

But there may still be times where a notion of "in context / out of context" serves a goal.

**Comment by scottpeterson** [comment]
Ultimately, I believe on Situation/MA/INT questions, any notion of Context or "out of context" should completely go away. 

Most of the time when a quizzer is in the wrong place, they give information we can deem "incorrect." That still applies. A quizzer cannot say incorrect information. 

But as long as they do not say incorrect information, why not let them quote from multiple contexts?

Hmmmm, as I write this, I see issues in implementation. Would a quizzer have to quote word perfect from "other" contexts?? Would they have to "demonstrate by stringing words together that they know what they're answering??" Thinking of a scenario where a quizzer quotes 15 words word perfectly from elsewhere in the material, one of which is THE WORD needed to be counted correct. Would they have to demonstrate "quoting of that single required word, in the midst of material from the correct context?"

It reminds of long answers, with clauses and connecting words, like "so" "if" "but". Technically if the quizzer doesn't say one of those, the meaning isn't fully there. This especially happens on "Therefore what?" Refs, where the entire rest of the verse is the answer. Do we need the quizzer to "answer the required material in a complete thought" ??

**Comment by gryphonshafer** [comment]
Agreed. Context for Q/F/CVRs should be the verses defined. CRs should be the chapter. All other context goes away. A quizzer is correct before they're incorrect. And if a quizzer ever says an incorrect answer, even if a single word, they're wrong.

But then that leaves the "does 1 word put the quizzer out of context" question. And if the above universe exists, I'm in favor of saying 1 word does put the quizzer out of context. It makes Q/F/Ref questions require more precision in answering, which would separate them in difficulty from INT/MA/SIT, which I think is a good thing because it rewards quizzers who are more prepared without making Quizzing as a whole too difficult for rookies to get into.

**Comment by jswingle** [comment]
@jttower @gryphonshafer @scottpeterson what do you think about closing this out and merging it with https://github.com/gryphonshafer/Quizzing-Rule-Book/issues/120 ?

**Comment by gryphonshafer** [comment]
Maybe. There's a necessary antecedent debate for both this and #120's initial posts, it seems to me. Should context be effectively eliminated for all but Q/F/Rs?

Or put a different way, assume the following becomes true:

> Context for Q/F/CVRs should be the verses defined. CRs should be the chapter. All other context goes away.

Is that better or worse than the current context rules, and why?

**Comment by jswingle** [comment]
I think the antecedent debate is necessary, sure, but my reason for wanting to merge the issues is that that they seem to be about exactly the same thing. This is a bit of a complex issue, though, so I wanted to double-check that others agreed with me they were the same issue before merging.

**Comment by jswingle** [comment]
And actually, since this issue was opened first, we should close out #120 and instead merge it with this one.

**Comment by jttower** [comment]
I think they are similar enough we can merge them.

**Comment by jswingle** [comment]
I've closed out issue #120 - please reference the entire discussion in that issue, as there's a lot that was said therein. https://github.com/gryphonshafer/Quizzing-Rule-Book/issues/120

**Comment by jswingle** [comment]
I've also closed out issue #116 to merge it with this issue, so please check out that discussion as well.

**Comment by 24RMiller** [comment]
I would support removing context on MAs, Ints, and Sits like @scottpeterson said. CVRs, Qs, and FTVs would remain with one verse context. CRs I could be persuaded either way. In theory, allowing the entire chapter as context could be beneficial, I just worry people could speed quote chapters too quickly for the quizmaster to hear what they're saying. Limiting CR context to the individual verse also could have its merit, but I am still undecided on where I stand.

I see 3 reasons for removing context on Ints, MAs, and Sits:

1. To lead to more correct answers
2. To make it more objective for officials
3. To reward quizzers that can quickly give multiple occurrences of the word or phrase jumped on

For a more in-depth reasoning let's look at an example from this past year. There was a 5 verse section in Ephesians 4 that had the phrase "do not." Thus, I decided that no matter what, if I jumped on "do not," I would quote those 5 verses. I still had a slim chance of getting it correct, but it gave me the best shot. I find it rather odd that I can essentially give 5 answers for what I jump on, but only because they are in an arbitrary 5 verse context. However, if there was another one 6 verses away, I could not give that as an answer.

To me, on an int or ma, the quizzer should be allowed to say anything that comes from the material. If a quizzer jumps on "do not," they can simply spout off as many do nots as they can in 30 seconds. This takes away much of the subjectivity as to whether a quizzer gave enough to be out of context. On situations, if there's multiple quotes that start the same way, you should be able to quote one, and if the quizmaster doesn't say "That's the quote" you just move to a different one. The incorrect answer rule still applies, so if a quizzer at any point provides incorrect information they are immediately wrong.

It always seems a little “unfair” when someone guesses the wrong answer of a 50/50 (and no, this is not in response to one specific event at Internationals this past year), when they gave material straight out of Scripture. If a quizzer can keep providing “correct information” there’s no reason they should be called wrong if they were more than 5 verses away from the answer, but given their full time if in context. I realize like Gryphon said this is a iceberg issue, but I do think it is something that should be looked at eventually.

**Comment by kclimenhaga** [comment]
Rather than making context go away entirely for these question types, could we consider limiting context to the chapter? I don't want to get into a situation where a quizzer is expected to jump on "Truly" in Luke and then quote all 6 "Truly I tell you" verses to get a question correct. (Yes, quizzers at the Internationals level would probably burn a "truly I tell you" question anyhow, but you get the point.) I do think that the 5-verse context rule probably goes back to the days before easily accessible concordances, so I see the value in doing away with that particular rule, but I don't think getting rid of context entirely is the right solution. I understand the "unfairness" of guessing the wrong answer on a 50/50, but I don't think it's any more unfair than when a quizzer jumping on syllables gets "Jesus" or a similarly common word.

**Comment by lukeshev** [comment]
My thoughts, coming at it from the perspective of Internationals:
- For Q, CVR, and F type questions, keep context to the single verse. Remove the “1 word can take you out of context” rule, as this can conflict with CVRs with one word answers, or Finishes with one word differences, etc.
- For everything else, I believe context should remain 5 verses before/after. Eliminating or expanding context altogether for any of these types would result in a shift in the dynamic of quizzing at Internationals. Quizzers could now jump much quicker and have a much higher chance at being called correct by quoting all passages with the spoken phrase. This would greatly diminish the skill required to jump at the right time, which I think is a huge part of quizzing that should be preserved at Internationals.

**Comment by Gpoulsen3hills** [comment]
add bullet point to 5.2.4 Incorrect.... if the quizzer quotes material out of context for that question type
